source/_integrations/plugwise.markdown

```diff
@@ -89,151 +89,118 @@ This integration follows standard integration removal. No extra steps are requir
 
 This will also remove all connected Adam devices (such as Anna, Tom or Lisa) or connected Adam/Stretch plugs.
 
-### Actions
+## Available actions
 
-#### Update Smile data
+The following actions, along with their examples, can be with the climate gateways:
 
-Forced update of data from your Smile can be triggered by calling the generic `homeassistant.update_entity` action with your Smile entity as the target.
+For all climate gateways:
 
-```yaml
-# Example script change the temperature
-script:
-  force_adam_update:
-    sequence:
-      - action: homeassistant.update_entity
-        target:
-          entity_id: climate.living_room
-```
-
-#### Reboot the Plugwise gateway
-
-action: `button.press`
-
-```yaml
-# Example script change the thermostat schedule
-script:
-  reboot_gateway:
-    sequence:
-      - action: button.press
-        target:
-          entity_id: button.adam_reboot
-```
-
-#### Set HVAC mode
-
-action: `climate.set_hvac_mode`
-
-Available options include `off` (Adam only) `auto`, `cool`, `heat`, and `heat_cool` (Anna with Elga only).
-
-The meaning of `off` is that the Adam regulation is set to off. This means that the connected HVAC-system does not heat or cool, only the domestic hot water heating function, when available, is active.
-
-The meaning of `cool` or `heat` is that there is no schedule active. For example, if the system is manually set to cooling- or heating-mode, the system will be active if the room temperature is above/below the thermostat setpoint.
-
-The meaning of `heat/cool` is that there is no schedule active. For example, if the system is in automatic cooling- or heating-mode, the active preset or manually set temperature is used to control the HVAC system.
-
-The meaning of `auto` is that a schedule is active and the thermostat will change presets/setpoints accordingly.
+- `climate.set_preset_mode`
```

Comment:
We want to avoid/remove calling them by their identifiers.

Additionally, we aim not to document features from entity components in the integration documentation.

Reply:
What direction would you like to take, I was trying to steer clear of copy/pasting the entity component (e.g., vicare) and wasn't sure if just listing them (e.g., daikin) would make much sense to the user without a little explanation?

Reply:
Well, that is a good question... 😄 

The point being, from a user perspective, `climate.set_preset_mode` is a technical detail, that cannot be used in the UI at all. So, for users needing this level of documentation, this is a mismatch.

But your question is valid. I wonder if we even should document this detail level on each integration page, and if we should, well I guess we should be able to link it up to the climate docs in a non-technical way (which we can't right now?)

@c0ffeeca7 Sorry for pulling you in here, but do you have any insights, direction, or insights on this one?

../Frenck

Reply:
Good point @frenck. I always wondered what the purpose is of these examples. If every integration shows these examples there is a lot of repetition going on.

Reply:
Reworked actions section

Reply:
Any suggestions (other than what I already rewrote) or should we create an issue out of this to pick up outside of a PR?